cfgov/unprocessed/apps/youth-employment-success/js/index.js

```diff
@@ -6,6 +6,11 @@ import routeOptionFormView from './route-option-view';
 import routeOptionToggleView from './route-option-toggle-view';
 import store from './store';
 
+Array.prototype.slice.call(
+  document.querySelectorAll( 'input' )
+).forEach( input => {
+  input.disabled = false;
```

Comment:
Do you want to be doing this, or do you want to be removing the attribute `input.removeAttribute( 'disabled' );`?

Reply:
yes, good catch